Multisig/Logic/Models/Chain.swift

```diff
@@ -415,6 +415,6 @@ extension Chain {
     }
 
     func isSupport(feature: Feature) -> Bool {
```

Comment:
isSupported would be more natural to read

Reply:
`isSupported(feature: Feature) -> Bool` or `supports(feature: Feature) -> Bool`